CHANGELOG.md

```diff
@@ -61,6 +62,7 @@
 ### ✅ Added
 
 ### ⚠️ Changed
```

Comment:
I think mentioning this only in one place, for `stream-chat-android-ui-components`. is enough.

Reply:
`DeletedMessageListItemPredicate` is located in UI Comp while `DeletedMessageVisibility` is in UI Common.
I agree that the one being deprecated should have the primary focus but I wanted to be thorough.